app/controllers/welcome_controller.rb

```diff
@@ -1,4 +1,6 @@
 class WelcomeController < ApplicationController
+  
+  skip_load_and_authorize_resource
```

Comment:
since you removed load and authorize_resource from application controller you don't need to skip it in here.